python/cucim/src/cucim/skimage/transform/_warps.py

```diff
@@ -774,21 +774,29 @@ def _clip_warp_output(input_image, output_image, mode, cval, clip):
         the image boundaries.
     clip : bool, optional
         Whether to clip the output to the range of values of the input image.
-        This is enabled by default, since higher order interpolation may
-        produce values outside the given input range.
+        If `order` > 1, this will be enabled by default, since higher order
+        interpolation may produce values outside the given input range.
 
     """
     if clip:
```

Comment:
Should we be doing something to handle `clip is None` before this?

Reply:
I refactored slightly now to move the clip is None handling inside of this function and remove the `if clip:` indentation here. There is still one instance where `clip=None` also still needs to be checked externally outside of this function.